notebooks/example.ipynb

```diff
@@ -200,6 +199,143 @@
     "%%bash\n",
     "cat ./fixtures/rendered_config.yaml"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ea5eff92",
+   "metadata": {},
+   "source": [
+    "## Example 3: Comparing two config files\n",
+    "Let's explore using the at using the `config.compare()` method to compare two two config files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71d27692",
+   "metadata": {},
+   "source": [
+    "We again need to start by importing the `uwtools.api.config` from the uwtools python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "id": "7fb40514",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from uwtools.api import config"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0103f44",
+   "metadata": {},
+   "source": [
+    "The uwtools.api.config.compare() method has the following syntax:\n",
+    "\n",
+    "`uwtools.api.config.compare(config_1_path: Path | str, config_2_path: Path | str, config_1_format: str | None = None, config_2_format: str | None = None)→ bool`\n",
+    "\n",
+    "Recognized file extensions are: ini, nml, sh, yaml.\n",
+    "\n",
+    "Parameters:\n",
+    "\n",
+    "- config_1_path – Path to 1st config file\n",
+    "\n",
+    "- config_2_path – Path to 2nd config file\n",
+    "\n",
+    "- config_1_format – Format of 1st config file (optional if file’s extension is recognized)\n",
+    "\n",
+    "- config_2_format – Format of 2nd config file (optional if file’s extension is recognized)\n",
+    "\n",
+    "The method returns False if config files had differences, otherwise True."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b121dc29",
+   "metadata": {},
+   "source": [
+    "For example, lets compare the two nml files that contain different text."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 42,
+   "id": "5dcff125",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "False"
+      ]
+     },
+     "execution_count": 42,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# path of nml file that contains 'greeting = \"Hello\" recipient = \"World\"'\n",
+    "file_a = \"./fixtures/config_test_file_a.nml\"\n",
+    "\n",
+    "# path of nml file that contains 'greeting = \"Hello\" recipient = \"Different World\"'\n",
+    "different_file_b = \"./fixtures/config_test_file_b.nml\"\n",
+    "\n",
+    "config.compare(file_a,different_file_b,None,None)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c16c38d0",
+   "metadata": {},
+   "source": [
+    "The config() method returns False to denote the files are different.\n",
```

Comment:
Another thing we could consider in these notebooks is enabling the `uwtools` logging, such that we get additional information when we use the tools. Logging can be enabled like this:
```
from uwtools.api.logging import use_uwtools_logger
use_uwtools_logger()
```
Then, for example:
```
>>> from uwtools.api.logging import use_uwtools_logger
>>> use_uwtools_logger()
>>> config.compare("ex3.1.nml", "ex3.2.nml")
[2024-07-17T20:44:29]     INFO - ex3.1.nml
[2024-07-17T20:44:29]     INFO + ex3.2.nml
[2024-07-17T20:44:29]     INFO ---------------------------------------------------------------------
[2024-07-17T20:44:29]     INFO foo:               n:  - 88 + 99
False
```
That's in a Python REPL, but behavior in a notebook cell should be similar. The input files `ex3.1.nml` and `ex3.2.nml` look like
```
&foo n=88, s="string" /
```
and
```
&FOO
  S = "string"
  N = 99
/
```
respectively.